modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -120,6 +121,15 @@ Configmaps "vmware-to-kubevirt-os" not found
 This warning does not affect the VMware virtual machine import.
 endif::[]
 
+* The following error message is displayed in the {product-title} console if a non-admin user tries to import a VM:
```

Comment:
Active voice be better. I would also rearrange this so that the user information leads the sentence. 
Eg: "If a non-admin user imports a virtual machine, the following error message displays:"

Reply:
If you do change to active voice, please also 's/is displayed/displays/' above in line 114

Reply:
Suggestion rejected. IBM Style Guide has no problem with "is displayed". See p. 229 and other examples.